zebrad/tests/common/config.rs

```diff
@@ -77,6 +77,6 @@ pub fn persistent_test_config() -> Result<ZebradConfig> {
 pub fn testdir() -> Result<TempDir> {
     tempfile::Builder::new()
         .prefix("zebrad_tests")
-        .tempdir()
+        .tempdir_in("/media/oxarbitrage/4eb53770-4e4f-4d80-a830-0914f3f5b89a/zebra/issue4350")
```

Comment:
TODO: create a ticket to make this argument an env variable. The `send_transaction_test` copies the full blockchain to this directory so it is very possible that the user running the test to be out of space in its "hda" drive.